pkg/virt-config/config-map.go

```diff
@@ -49,6 +49,7 @@ const (
 	useEmulationKey        = "debug.useEmulation"
 	imagePullPolicyKey     = "dev.imagePullPolicy"
 	migrationsConfigKey    = "migrations"
+	cpuModelKey            = "default-cpu-model"
```

Comment:
ugh, I never notices that we have a mixture of `-` and camel case keys.

Reply:
:)
That's one of the benefits of having all those placed together - I'll change

Reply:
your style is right now the common one. I don't think that you have to change anything. Just noticed it.